processor/migration/emphasis.ts

```diff
@@ -27,24 +27,40 @@ const trimEmphasis = (node: Emphasis | Strong, index: number, parent: Parent) =>
   let trimmed = false;
 
   visit(node, 'text', (child: Text) => {
-    const newValue = child.value.trim();
+    const newValue = child.value.trimStart();
```

Comment:
~~Why do we have to trim the start and end separately?~~ Oh I see now! In cases where there are multiple nodes!